.nvmrc

```diff
@@ -0,0 +1 @@
+5.3.0
```

Comment:
I don't think 5.3.0 is the right version right now. Both the CI and `vagrant-wagtail-develop` use node 4, so I'd rather have an `.nvmrc` that says "4" as well. Using multiple major node/npm versions across the infrastructure creates complexities when dealing with shrinkwraps for example.

Once this is in place, it would make sense to migrate everything to node 6/7 (and npm 3/4), but I'd rather not tie having the nvmrc to that.